.github/workflows/tests.yaml

```diff
@@ -0,0 +1,58 @@
+name: Tests
+
+on: [push, workflow_dispatch]
+
+jobs:
+  fmtcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: make fmtcheck
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.18.0
```

Comment:
Not sure that it matters too much, but I think Vault 1.10 is still on go 1.17.x

Reply:
Good point, may as well match it up for the release branch